src/generator/templates/sample.njk

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*! THIS FILE IS AUTO-GENERATED */
+
+const {google} = require('googleapis');
+const {{api.name}} = google.{{api.name}}('{{api.version}}');
+
+{% if method.description %}
+/*
+ * {{ method.description|cleanComments|safe|replace("\n", "\n * ") }}
+ */
+{% endif %}
+
+async function main() {
+  // By default, this method will look for, in order:
+  // 1. An environment variable set to `GOOGLE_APPLICATION_CREDENTIALS`
+  //    pointing to a service account credential file
+  // 2. A GCE metadata server, present in Google Cloud products like
+  //    Compute Engine, Kubernetes Engine, Cloud Run, etc
+  // 3. A local OAuth token written by the Cloud SDK, obtained by running
+  //    `gcloud auth application-default login`.  This is preferred for local
+  //    development.
+  const auth = new google.auth.GoogleAuth({
+    // Scopes can be specified either as an array or as a single, space-delimited string.
+    scopes: [
+      {% for scope in method.scopes %}'{{scope}}',
+      {% endfor %}
+    ],
+  });
+
+  // Acquire an auth client, and bind it to all future calls
+  const authClient = await auth.getClient();
+  google.options('auth', authClient);
+
+  // Do the magic
+  const res = await {{method.id}}({
+    {% if method.parameters %}
+    {% for pname, p in method.parameters|dictsort %}
+    {% if p.description %} // {{ p.description|oneLine|cleanComments|safe }}{% endif %}
```

Comment:
I don't really like `oneLine`, I see nothing bad if comments spread into multiple lines, like we do in `jsdoc`.

Reply:
Tend to agree.  Fixed!